runtime/compiler/env/J9SharedCache.cpp

```diff
@@ -100,47 +100,33 @@ TR_YesNoMaybe TR_J9SharedCache::isSharedCacheDisabledBecauseFull(TR::Compilation
    return _sharedCacheDisabledBecauseFull;
    }
 
-bool
-TR_J9SharedCache::initCCVCaching()
+CCVResult
+TR_J9SharedCache::getCachedCCVResult(TR_J9VMBase *fej9, TR::CompilationInfo *compInfo, TR_OpaqueClassBlock *clazz)
    {
-   if (!_classChainValidationMutex)
+   if (TR::Options::getCmdLineOptions()->allowRecompilation()
```

Comment:
I did add the test in bootstrap here 
https://github.com/eclipse/openj9/blob/cf5065d17d6880cd4bd988d23d91ca6490008e07/runtime/compiler/control/DLLMain.cpp#L438-L444
but I suppose this was an additional test just in case it's possible for recomp to get disabled at runtime.